boards/arm/holyiot_yj16019/holyiot_yj16019_defconfig

```diff
@@ -6,8 +6,5 @@ CONFIG_BOARD_HOLYIOT_YJ16019=y
 # Enable MPU
 CONFIG_ARM_MPU=y
 
-# Enable RTT
-CONFIG_HAS_SEGGER_RTT=y
-
```

Comment:
This should probable be `CONFIG_USE_SEGGER_RTT=y` since this board has no physical pins for a UART for console.

Reply:
Fixed. See new PR comment.